src/select-menu/src/SelectMenu.js

```diff
@@ -128,6 +138,40 @@ export default class SelectMenu extends PureComponent {
     return {}
   }
 
+  getTitleView = (close, title, headerHeight, titleView) => {
```

Comment:
I think you could get rid of this code in favor pushing it into the component that actually uses `titleView` instead.